pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -52,6 +52,7 @@ import (
 const (
 	ProviderName     = "openstack"
 	AvailabilityZone = "availability_zone"
+	defaultTimeOut   = 60
```

Comment:
If this is seconds, it should be `time.Duration(60*time.Second)` Right?

https://play.golang.org/p/gmIFXzUgQF

Reply:
yup indeed